### PR TITLE
fix copy script

### PR DESCRIPTION
**Pull request by tac0turtle**

### CODE REVIEW

File: .github/workflows/copy-md.yml [hunk 1]
```diff
@@ -13,13 +16,13 @@ jobs:
...
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
-          node-version:
+          node-version: 16
 
       - name: Install Dependencies
         run: |
-          npm install -g jq   
+          yarn install -g jq
```
Comment (julienrbrt) [inline .github/workflows/copy-md.yml:25]: Why not installing using `apt install jq` instead of using npm?

Comment (tac0turtle) [reply .github/workflows/copy-md.yml]: either way

Comment (julienrbrt) [reply .github/workflows/copy-md.yml]: Seems like yarn had the same issue than npm: https://github.com/cosmos/cosmos-sdk-docs/actions/runs/6148633384/job/16682750925?pr=28, so hopefully apt will work.

Comment (julienrbrt) [reply .github/workflows/copy-md.yml]: I think `sudo apt-get install jq -y` is better than `sudo apt install jq -y` looking at the log, as `apt install` isn't stable (TIL).

Comment (tac0turtle) [reply .github/workflows/copy-md.yml]: jq is already there, https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md. no need for it, just found this

Comment (julienrbrt) [reply .github/workflows/copy-md.yml]: lol. better :)

**Review by julienrbrt — APPROVED** [review]
utACK for having the script run.
However, the result isn't pushed anywhere, so the action should be updated.

**Comment by tac0turtle** [comment]
> utACK for having the script run. However, the result isn't pushed anywhere, so the action should be updated. This action works well for that for instance: https://github.com/julienrbrt/simapp/blob/d565eb8/.github/workflows/refresh.yml#L35C1-L38

HAHAH, i was confused as well. ill make the changes

**Pull request merged**